### DISCUSSION TITLE
sorting-spec in folder note are not visible in property editor. Did I made a mistake or it’s a bug ?

### DISCUSSION THREAD

**Discussion by LordPhoenixfr** [SebastianMC/obsidian-custom-sort#168] [Q&A]
Hi folks, 
I’m using sortspec file in root folder to organize general sort of my vault and I add specific sort criteria for some folders in folder note properties but whereas I see sorting-spec properties from sortspec file in obsidian properties editor  I don’t see the properties from folder note. And if I want use the properties editor to add one property sorting is no longer functional. 

Did I miss something somewhere or it’s a bug ?

**Comment by SebastianMC** [comment]
Hi @LordPhoenixfr 

based on my (not good) experience with Obsidian properties editor for various scenarios, we need to wait some more iterations, improvements and Obsidian releases to get a mature, reliable, user-friendly, polished UX, maybe configurable, properties editor.
Because of the above, I switch to 'source mode' whenever it comes to editing sorting specification in a note (including folder notes).

Did you try the 'source mode'? It can be enabled in the 'View' menu or via the editor mode icon in the status bar. When enabled, you can conveniently edit the YAML frontmatter (aka 'properties') in the old way with a fully fledged plain-text editor.

Regarding the difference between regular notes and folder notes - I think it could be related to a broken folder notes plugin, which you use. There are three or four available and part of them are not compatible with newest Obsidian releases. Obsidian team tends to bring more and more breaking changes with each release in a not backward-compatible way, thus breaking some of the existing plugins. Recently I've been forced to abandon the Aidenlx' Folder Notes and switch to another Folder Notes plugin (https://github.com/LostPaul/obsidian-folder-notes).
